README.md

```diff
@@ -1,5 +1,9 @@
 # dart_to_js_script_rewriter
 
+[![Pub](https://img.shields.io/pub/v/dart_to_js_script_rewriter.svg)](https://pub.dartlang.org/packages/dart_to_js_script_rewriter)
+[![Build Status](https://travis-ci.org/Workiva/dart_to_js_script_rewriter.svg?branch=travis-ci)](https://travis-ci.org/Workiva/dart_to_js_script_rewriter)
```

Comment:
link to docs is already available on pub in the right side of the screen
<img width="1024" alt="screen shot 2016-02-03 at 11 47 54 am" src="https://cloud.githubusercontent.com/assets/3778206/12791291/16284518-ca6c-11e5-8b7b-17219768eb2e.png">